api/v1alpha1/volumereplication_types.go

```diff
@@ -39,14 +39,14 @@ const (
 type State string
 
 const (
-	// Replicating means the volume is mirroring or replicating
-	Replicating State = "Replicating"
+	// PrimaryState represents the Primary replication state
+	PrimaryState State = "Primary"
 
-	// ReplicationFailure means the last replication operation failed
-	ReplicationFailure State = "Failed"
+	// SecondaryState represents the Secondary replication state
+	SecondaryState State = "Secondary"
 
-	//Resyncing means that the volume is resyncing
-	Resyncing State = "Resyncing"
+	// UnknownState represents the Secondary replication state
```

Comment:
It should be 

// UnknownState represents the Uknown replication state

Reply:
fixed.